ngrinder-controller/src/main/java/org/ngrinder/script/service/ScriptValidationService.java

```diff
@@ -112,7 +113,7 @@ public String validate(User user, IFileEntry scriptIEntry, boolean useScriptInSV
 						StringUtils.defaultIfBlank(scriptEntry.getEncoding(), "UTF-8"));
 			}
 			File doValidate = localScriptTestDriveService.doValidate(scriptDirectory, scriptFile, new Condition(),
-					config.isSecurityEnabled(), hostString, getTimeout());
+					config.isSecurityEnabled(), config.getControllerProperties().getProperty(PROP_CONTROLLER_SECURITY_LEVEL, ""), hostString, getTimeout());
```

Comment:
how about create getSecurityLevel method in config class??
config.getControllerProperties().getProperty(PROP_CONTROLLER_SECURITY_LEVEL, "") => config.getSecurityLevel()